AppCenterAnalytics/AppCenterAnalyticsTests/MSAnalyticsTests.m

```diff
@@ -311,7 +312,29 @@ - (void)testTrackEventWhenAnalyticsDisabled {
   OCMVerifyAll(channelUnitMock);
 }
 
-- (void)testTrackEventWhenTransmissionTargetDisabled {
+- (void)testTrackEventWithTypedPropertiesNilWhenAnalyticsDisabled {
+
+  // If
+  id analyticsMock = OCMPartialMock([MSAnalytics sharedInstance]);
```

Comment:
This  (318-327) should be refactored out into a setup method because it's essentially repeated for 6 tests with little change.